node/src/sub_lib/utils.rs

```diff
@@ -285,7 +278,7 @@ pub mod tests {
 
         TestLogHandler::new().exists_no_log_containing(&format!(
             "handle_ui_crash_message_does_not_crash_if_not_crashable: {}",
-            BEGINNING_OF_CRASH_RQ_MESSAGE
+            "Received a crash request"
```

Comment:
put the "variable" directly in to the leading string, as continuation of it.

Reply:
Done!